### ISSUE TITLE
Theme not working in gtk2 based apps

### ISSUE THREAD

**Issue by rudrab** [nana-4/materia-theme#568]
Hi,
There are still some apps, e.g. pidgin, which is using gtk2. The problem is as materia-theme now uses meson build, which installs the theme in `~/.local/share`, the location is invisible to `pidgin`. 
Now, I have installed the materia theme using `meson`, and then copied it to ~/.themes again to make it visible to pidgin.

Is there any other way for gtk2 based apps?

**Comment by nana-4** [comment]
Hi, it seems GTK 2 themes aren't loaded from `~/.local/share/themes`. [Arc theme's installation doc](https://github.com/jnsh/arc-theme/blob/master/INSTALL.md#note-about-installation-in-users-home-directory) has a note about that.

If I remember correctly, some other themes, on the other hand, aren't loaded from `~/.themes`. So I'd recommend installing the theme under the default `/usr`, unless you have a particular reason.

**Comment by actionless** [comment]
and which apps ca'nt see the theme under `~/.themes/` ?

**Comment by rudrab** [comment]
@actionless as told in original post, pidgin is such an app.

**Comment by actionless** [comment]
are you sure you answering the same question as i asked?

**Comment by nana-4** [comment]
@actionless At least [budgie-desktop can't/couldn't](https://github.com/solus-project/budgie-desktop/issues/1202). Maybe there are/were more, but I don't remember others :/

**Comment by rudrab** [comment]
> are you sure you answering the same question which i asked?

@actionless no...i am sorry. `Pidgin` can actually see `~/.themes`, not `~/.local/share/themes`. Sorry for that.

**Comment by actionless** [comment]
hm, i see they wrote:

>>> Mutter/GTK started to break ~/.themes some time a

would be interesting to know more

if that thing gonna broaden probably in change_color.sh (or inside meson itself?) to move gtk2 theme under ~/.themes and the rest under .local/share

**Comment by actionless** [comment]
btw, quite a low move (as usual :D) to change default theme location as soon as gtk2 development got officially frozen

**Comment by actionless** [comment]
btw, also gonna inter-link this here: https://github.com/themix-project/oomox-gtk-theme/issues/45

**Comment by nana-4** [comment]
By default our meson installs the theme under `/usr/share/themes`, so this would be a `change_color.sh` specific issue :)

**Comment by nana-4** [comment]
Personally I don't think it's great that some themes are in `~/.themes` and others are in `~/.local/share/themes`. Casual users would be confused, especially when uninstalling the themes.

IMO it would be the best to place all themes under `/usr/share/themes`, the safest theme location for every app. I'm not sure you'll agree with it as an Oomox author though.

**Comment by actionless** [comment]
themix have the code already for elevating permissions to write outside of /home/ (used for spotify theme)

but i think many of users wouldnt like if it will be un-alternatively writing to /usr/
